README.md

```diff
@@ -11,4 +11,11 @@ This adds an `autocomplete="off"` attribute to password inputs on the signup and
 Note that this feature is generally ignored by modern web browsers due to the inherent insecurity of attempting to prevent password managers from working which encourages weak passwords or bad password management practices (like using sticky notes).
 See [Choosing Secure Passwords](https://www.schneier.com/blog/archives/2014/03/choosing_secure_1.html) for more details.
 
-Check out the [wiki page](https://wiki.jenkins.io/display/JENKINS/Paranoia+Plugin) for more information.
+## Enable X-XSS-Protection Header
+
+This feature enables the HTTP header `X-XSS-Protection: 1; mode=block` to be sent on all requests which some web browsers support as a way to automatically block suspected cross-site scripting attacks.
+Several web browsers (e.g., Firefox, Edge) do not support this header, though enabling it may still help mitigate unknown XSS issues in browsers that do support it.
```

Comment:
```suggestion
Several web browsers (e.g., Firefox, Edge, and Chrome) do not support this header.
```
Yeah, I know Chrome still includes support for it, but they've announced they're going to remove it and their implementation is so broken anyway it's hard to say they support it. Given the inherent problems in this header, I think saying it may help or mitigate anything is unsubstantiated.

Reply:
Yeah, good point.